classes/Attribute.php

```diff
@@ -19,20 +19,26 @@
  * needs please refer to http://www.prestashop.com for more information.
  *
  * @author    PrestaShop SA <contact@prestashop.com>
- * @copyright 2007-2015 PrestaShop SA
+ * @copyright 2007-2016 PrestaShop SA
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+/**
+ * Class AttributeCore
+ */
 class AttributeCore extends ObjectModel
 {
     /** @var int Group id which attribute belongs */
     public $id_attribute_group;
 
     /** @var string Name */
     public $name;
+    /** @var string $color */
     public $color;
+    /** @var int $position */
     public $position;
+    /** @todo Find type */
     public $default;
```

Comment:
because it's public we cant remove it, if a module use it, the module will be broken.

To me we **should't** remove it now, because else we will have to keep it until PrestaShop 1.8 /c @Shudrum